code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -305,7 +301,8 @@ GLOBAL_VAR(medibot_unique_id_gen)
 	if(emagged == 2) //Everyone needs our medicine. (Our medicine is toxins)
 		return TRUE
 
-	if(HAS_TRAIT(C,TRAIT_MEDIBOTCOMINGTHROUGH) && !HAS_TRAIT_FROM(C,TRAIT_MEDIBOTCOMINGTHROUGH,medibot_counter)) //someone is healing them already sweetie
+	if(HAS_TRAIT(C,TRAIT_MEDIBOTCOMINGTHROUGH)) //romance of the three medbots
+		REMOVE_TRAIT(C,TRAIT_MEDIBOTCOMINGTHROUGH,TRAIT_MEDIBOTCOMINGTHROUGH) //casus belli

```

Comment:
These are funny, but surely we should have actual meaningful comments as to what this does?